### ISSUE TITLE
Прошу помочь в установке... на убунте

### ISSUE THREAD

**Issue by Lsdandk** [litnimax/astconfman#27]
Если живые есть, распишу подробности... по инструкции которая в описании оно не запускается

**Comment by litnimax** [comment]
Т.е. я помогу а ты инструкцию напишешь?

**Comment by Lsdandk** [comment]
Ну готов и написать. Тут сложностей нет… языком владею…

**Comment by litnimax** [comment]
Ну ок,  что не получается?

**Comment by Lsdandk** [comment]
Делаю ./run.py 

Получаю: [http://smylink.ru/cbm2.png](url)

**Comment by litnimax** [comment]
Ну так и шо? Это же warnings.
Дальше все стартануло?
На локалхост заходили?

**Comment by Lsdandk** [comment]
Так... ставил на серверную ос, без браузеров и прочего. Пробую ставить на декстоп...
